config.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "Helm",
+  "type": "executer",
+  "imgUrl": "icon.png",
+  "execProgram": "node",
+  "main": "app.js",
+  "version": "1.1.0",
```

Comment:
I think we should change the version to 2.0.0. The plugin is no longer compatible with the 1.0.6 version, `helmInstall` method's name and parameters' names have changed, so every pipeline that uses this pipeline won't work correctly.

Reply:
Fixed